mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeansModel.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.spark.rdd.RDD
 @Since("1.6.0")
 @Experimental
 class BisectingKMeansModel @Since("1.6.0") (
-    @Since("1.6.0") val root: ClusteringTreeNode
+    @Since("1.6.0")private[clustering] val root: ClusteringTreeNode
```

Comment:
If we want to hide `root`, we should also hide class `ClusteringTreeNode` entirely because it is not used anywhere else.